app/src/main/java/pl/agh/eye/DetectionUtils.kt

```diff
@@ -0,0 +1,90 @@
+package pl.agh.eye
+
+import org.opencv.core.*
+import org.opencv.imgproc.Imgproc
+import org.opencv.objdetect.CascadeClassifier
+
+class DetectionUtils {
+    fun detect(classifier: CascadeClassifier?, img: Mat?): MatOfRect {
+        val elems = MatOfRect()
+        classifier?.detectMultiScale(img, elems, 1.3, 5)
+        return elems
+    }
+
+    fun getEyeGazeDirectionEdges(eyeMat: Mat?, threshold: Double): Point {
+        val browlessEye = Mat(
+            eyeMat,
+            Rect(0, eyeMat!!.height() / 4, eyeMat.width(), eyeMat.height() / 2)
```

Comment:
W sumie spoko by było wiedzieć co te liczby oznaczają, te 4, 2 i poniżej te 5.0, 70.0 i 3 itd. Nie musisz ich wyciągać do zmiennych bo ich jest w cholere, ale może jakiś komentarz by był ok. Chyba, że to po prostu jakieś wartości z algorytmu są i nie wiadomo czemu akurat takie są?